package.json

```diff
@@ -14,9 +15,9 @@
     "web-vitals": "^1.1.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
```

Comment:
Keep in mind that, this project required making use of the `create-react-app` script.

You might want to add it back, thanks.

Reply:
Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.